### DISCUSSION TITLE
Chatwoot Moodle plugin (already completed)

### DISCUSSION THREAD

**Discussion by Buda9** [chatwoot/chatwoot#9988] [Show and tell]
Hey guys,

I created Moodle plugin for Chatwoot. You can find it [here](https://github.com/Buda9/moodle-local_chatwoot). Plugin automatically "collects" users first and last name, email and phone number.

Plugin settings page:

![Screenshot_269](https://github.com/user-attachments/assets/d1e71b63-e6da-4f05-b6e4-562793acf2d4)

![Screenshot_270](https://github.com/user-attachments/assets/f46572ff-6d8f-4e62-9c35-37c648344de0)

![Screenshot_271](https://github.com/user-attachments/assets/b1a43ef3-aee9-48c0-80ee-9c3c67b93d9c)

Hopefully it will be useful to someone, except me of course :)

**Comment by sojan-official** [comment]
This looks really great 😍 . Thanks a lot for sharing !!!
